j2objc.gradle

```
@@ -857,6 +857,125 @@ class J2objcXcodeTask extends DefaultTask {
+          
+          // include all resources in a temporary folder
+          String j2objcResourceDirName = "j2objcResources"
+          String j2objcResourceDirPath = "${project.buildDir}/${j2objcResourceDirName}"
+          project.delete j2objcResourceDirPath
+          project.copy {
+              from project.files(
+                      project.fileTree(dir: "${project.projectDir}/src/main/resources",
+                        include: "**/*.*"))
+              into j2objcResourceDirPath
+          }  
+          
+          // create the podspec
+          def j2objcHome = project.j2objcConfig.j2objcHome
+          
+          def podName = "${project.name}"
+          def podspecName = "${podName}.podspec"
+          def podVersion = "${project.version}"
+          File podspecFile = new File(project.buildDir, "${podspecName}")
+          podspecFile.write("")
+          podspecFile.append(
+              "Pod::Spec.new do |s|"+lineSeparator+
+              "s.name = '${podName}'"+lineSeparator+
+              "s.version = '${podVersion}'"+lineSeparator+
+              "s.summary = 'Generated by the j2objc.gradle plugin.'"+lineSeparator+
+              "s.source_files = 'j2objc/*.{h,m}'"+lineSeparator+
+              "s.public_header_files = 'j2objc/*.h'"+lineSeparator+
+              "s.resources = '${j2objcResourceDirName}/**/*.*'"+lineSeparator+
+              "s.requires_arc = true"+lineSeparator+
+              "s.libraries = 'ObjC', 'guava', 'javax_inject', 'jre_emul', 'jsr305', 'z', 'icucore'"+lineSeparator+
+              "s.xcconfig = { 'HEADER_SEARCH_PATHS' => '${j2objcHome}/include', 'LIBRARY_SEARCH_PATHS' => '${j2objcHome}/lib' }"+lineSeparator+
+              "end")
+          
+          // link the podspec in pod file
+          File podFile = new File(xcodeProjectDir, "Podfile")
+          if (!podFile.exists()) {
+              println "No podfile doese exxist execute 'pod init' first"
```

Should this throw an error rather than just print and potentially fail silently? Can `pod init` be run automatically?